packages/jellyfish-api-core/__tests__/category/account/withdrawFutureSwap.test.ts

```diff
@@ -0,0 +1,922 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import BigNumber from 'bignumber.js'
+import { Testing } from '@defichain/jellyfish-testing'
+import { FutureSwap } from 'packages/jellyfish-api-core/src/category/account'
+import { RpcApiError } from '@defichain/jellyfish-api-core'
+
+const container = new MasterNodeRegTestContainer()
+const testing = Testing.create(container)
+let collateralAddress: string
+let oracleId: string
```

Comment:
These variables

 let oracleId: string
 const futInterval = 25
 const futRewardPercentage = 0.05
  
  should move inside setup function